src/pollution_model/run_medusa_2.py

```diff
@@ -622,12 +625,69 @@ def main(selected_polygon_gdf: gpd.GeoDataFrame,
     rainfall_event = MedusaRainfallEvent(antecedent_dry_days, average_rain_intensity, event_duration, rainfall_ph)
 
     # Run the pollution model
-    scenario_id = run_pollution_model_rain_event(engine, area_of_interest, rainfall_event)
+    scenario_id = run_pollution_model_rain_event(
+        engine, area_of_interest, rainfall_event
+    )  # pylint: disable=unsupported-assignment-operation
+
```

Comment:
I don't think these changes are needed, my pylint didn't fail here
```suggestion
    scenario_id = run_pollution_model_rain_event(engine, area_of_interest, rainfall_event)
```